tools/sgen/input/FluentUIStyle.yml

```diff
@@ -285,9 +288,13 @@ MSFPrimaryHeaderTokens extends MSFHeaderFooterTokens:
   textFont: $Typography.headline
   topPadding: $Spacing.medium
 
-AP_MSFDrawerTokens:
+AP_DrawerTokens:
```

Comment:
Why is the name being changed from MSFDrawerTokens to DrawerTokens?

Reply:
My understanding is that MSF prefix is only used for vNext components to avoid namespace collision.